### PR TITLE
Feature/#169

### PR DESCRIPTION
**Pull request by bvanderstee**
| Q             | A
| ------------- | ---
| Branch        | master for features and deprecations
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Documented?   | yes
| Fixed tickets | #169

This PR will make it possible to run shell commands and scripts via the shell task

### CODE REVIEW

File: src/Task/Shell.php [hunk 1]
```diff
@@ -75,12 +80,12 @@ public function run(ContextInterface $context)
     }
 
     /**
-     * @param $script
+     * @param array $scriptArguments
      */
-    private function runShell($script)
+    private function runShell(array $scriptArguments = array())
```
Comment (veewee) [inline src/Task/Shell.php]: You can drop the default value, it will always be an array here.

File: doc/tasks/shell.md [hunk 2]
```diff
@@ -18,7 +18,12 @@ parameters:
 
 This options specifies the paths to your shell scripts.
 You can specify one or more scripts. 
-All scripts need to succeed for the task to complete.
+
+You also can specify one or more shell commands:
+- In a string ``"./bin/command arg1 arg2 ... arg-n"``
+- In an array ``["./bin/command", "arg1", "arg2", ["arg-n"]]``
+
+All scripts / shell commands need to succeed for the task to complete.
```
Comment (veewee) [inline doc/tasks/shell.md]: Can you add a real configuration example for both underneath?

File: src/Task/Shell.php (hunk 1 above)
Comment (bvanderstee) [inline src/Task/Shell.php]: Thanks for the tip, default value has been removed.

File: doc/tasks/shell.md (hunk 2 above)
Comment (bvanderstee) [inline doc/tasks/shell.md]: Added configuration examples for all possible configurations.

**Pull request merged**